### PR TITLE
am(cors): swagger updated for 3.18.x

### PR DESCRIPTION
**Pull request by ashraf706**
Updated swagger.json for AM 3.18.x

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/am-idp-update-swagger-3-18/index.html)

### CODE REVIEW

File: am/current/management-api/3.18/swagger.json [hunk 1]
```diff
@@ -1 +1 @@
...ype":"string"},"domain":{"type":"string"},"createdAt":{"type":"string","format":"date-time"},"updatedAt":{"type":"string","format":"date-time"},"system":{"type":"boolean"},"claims":{"type":"array","items":{"type":"string"}},"expiresIn":{"type":"integer","format":"int32"},"discovery":{"type":"boolean"},"parameterized":{"type":"boolean"}}},"NewScope":{"type":"object","required":["description","key","name"],"properties":{"key":{"type":"string"},"name":{"type":"string"},"description":{"type":"string"},"iconUri":{"type":"string"},"expiresIn":{"type":"integer","format":"int32"},"discovery":{"type":"boolean"},"parameterized":{"type":"boolean"}}},"UpdateScope":{"type":"object","required":["description","name"],"properties":{"name":{"type":"string"},"description":{"type":"string"},"expiresIn":{"type":"integer","format":"int32"},"discovery":{"type":"boolean"},"parameterized":{"type":"boolean"},"iconUri":{"type":"string"}}},"PatchScope":{"type":"object","properties":{"name":{"type":"string"},"description":{"type":"string"},"iconUri":{"type":"string"},"expiresIn":{"type":"integer","format":"int32"},"discovery":{"type":"boolean"},"parameterized":{"type":"boolean"}}},"Reporter":{"type":"object",
```
[the remark this answers is not in the document]
Comment (ashraf706) [reply am/current/management-api/3.18/swagger.json:1]: The version should be 3.18.0, not the release version. Do you see then SNAPSHOT? If yes then it is wrong.
Our API references are not targeted to bug fix release:
https://docs.gravitee.io/am/current/am_devguide_management_api_documentation.html

[the remark this answers is not in the document]
Comment (passionne) [reply am/current/management-api/3.18/swagger.json:1]: Yes, the first line of the file is:

```
"swagger":"2.0","info":{"version":"3.18.17-SNAPSHOT","title":"Gravitee.io - Access Management API"},
```

[the remark this answers is not in the document]
Comment (ashraf706) [reply am/current/management-api/3.18/swagger.json]: doc regenerated with 3.18.17 which omits the snapshot

**Review by passionne — APPROVED** [review]
well done

**Pull request merged**